### PR TITLE
Add cloudbuild.yaml for Prow

### PR DESCRIPTION
**Pull request by namrata-ibm**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**: The Prow job fails due to missing cloudbuild.yaml

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE  
```

### CODE REVIEW

**Comment by namrata-ibm** [comment]
cc @pohly

**Review by pohly — APPROVED** [review]
looks good to me

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+release-tools/cloudbuild.yaml
```
Comment (msau42) [inline cloudbuild.yaml:1]: do we also need cloudbuild.sh?

Comment (namrata-ibm) [reply cloudbuild.yaml:1]: Thanks, PTAL.

**Pull request merged**